ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3SubsystemAdd.java

```diff
@@ -612,4 +614,29 @@ private static void addClusteringServices(final OperationContext context, final
             builder.setInstance(new SingletonBarrierService(policy)).setInitialMode(ServiceController.Mode.ON_DEMAND).install();
         }
     }
+
+    public final class ValueService<T> implements Service<T> {
```

Comment:
this may be `private static` class, if not used elsewhere.

Do you also plan to update this deprecated `Service` class to `org.jboss.msc.Service`?

Reply:
Good catch @chengfang - I fixed it. There is no plan to migrate to o.j.m.Service now. It will be handled in the future.